### PR TITLE
feat!: server technology changed to Spring

### PR DESCRIPTION
**Pull request by Sciator**

### CODE REVIEW

**Comment by Sciator** [comment]
Does server/maven related files (pom.xml, mvnw, .mvn ...) have to be in root dir ?

**Comment by Sciator** [comment]
Delete/replace ui/src/common/interface/*.ts files and PR is ready for merge

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip
```
[the remark this answers is not in the document]
Comment (kejmUP) [reply .mvn/wrapper/maven-wrapper.properties:1]: No changes should be done about Maven Wrapper. I know it's not standard to have JAR and other binary files inside the repo, but this is the best practice with the Maven Wrapper. More info: https://github.com/takari/maven-wrapper.

**Comment by kejmUP** [comment]
> Does server/maven related files (pom.xml, mvnw, .mvn ...) have to be in root dir ?

Yes, they have.

**Comment by kejmUP** [comment]
Closing, already in master.

**Pull request closed without merging**